dnsapi/dns_mydevil.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env sh
+
+# MyDevil.net API (2019-02-03)
+#
+# MyDevil.net already supports automatic Let's Encrypt certificates,
+# except for wildcard domains.
+#
+# This script depends on `devil` command that MyDevil.net provides,
+# which means that it works only on server side.
+#
+# Author: Marcin Konicki <https://ahwayakchih.neoni.net>
+#
+########  Public functions #####################
+
+#Usage: dns_mydevil_add   _acme-challenge.www.domain.com   "XKrxpRBosdIKFzxW_CT3KLZNf6q0HG9i01zxXp5CPBs"
+dns_mydevil_add() {
```

Comment:
the name should be `dns_deveil`,  not `dns_mydeveil`,  please update all the names.

Reply:
@Neilpang host name is mydevil.net and they are the ones who work on `devil` tool (AFAIK it's a set of custom scripts and it's not published anywhere). You can read more about it at: https://translate.google.com/translate?sl=pl&tl=en&u=https%3A%2F%2Fwiki.mydevil.net%2FDevil

Unless there's some other host, that uses the same exact tool (i do not know about any) it's probably better to keep the name as `mydevil` - just in case there is another `devil` API somewhere out there, that's completely different.

I cannot guarantee that it will work on any other host - i tested it only on mydevil.net.

Reply:
alright,  I didn't noticed that.